Makefile

```diff
@@ -1,15 +1,26 @@
 CC = gcc
-CFLAGS = -g -Wall -pedantic -std=c99 # -DDEBUG
 
-EXE = fib fib2 fib3 counter counter2 clock onetwo simpleadd
+TIME_DRIVER ?= simulation
+
+ifeq ($(TIME_DRIVER), linux)
+	CSTANDARD = gnu99
```

Comment:
Note that the linux timing api is only available as a gnu extension.

Reply:
Is this the timing API that includes nanosleep etc? Seems odd that it's tied to a compiler extension instead of an operating system/hardware platform.